src/types/soundPack.ts

```diff
@@ -23,6 +24,7 @@ export { defaultSoundPack };
 
 const ztmPoznańSoundPack: soundPack = {
   name: "ZTM Poznań",
+  volumeOverride: 0.2,
```

Comment:
The name doesn't clearly indicate neither data type, nor unit (fraction, percentage or other). This, combined with plain `1` being used in other files, is very confusing.

Reply:
The value itself indicates both. Volume is numeric - and if it's a .2, then obviously that must mean the scale is 0-1. It wouldn't make sense if it was anything else - on a scale of 0-100, this kind of granularity would be unnoticable to most any ear (and in strongly typed languages it would be impossible, since we'd be mixing floats and ints). Plus, it's being fed directly into the volume functions of the libraries we're using, both of which use a 0-1 float.